app/routes/order/international_order.js

```diff
@@ -0,0 +1,19 @@
+import Ember from "ember";
+import AuthorizeRoute from "./../authorize";
+
+export default AuthorizeRoute.extend({
+  orderService: Ember.inject.service(),
+
+  async model() {
+    return {
```

Comment:
no need to use `async-await` in model. Model will automatically take care of it.

Reply:
I am getting a promise , so i am using async-await

Reply:
`model` resolves promises automatically no need to explicitly resolve it :)

Reply:
okk... i missed that point